modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,209 @@
+Currently supported authentication methods are basic authentication (`basicAuth`) and client TLS (`tlsConfig`) authentication.
++
+* The following example configures basic authentication:
++
+[source,yaml]
+----
+basicAuth:
+  username:
+    <usernameSecret>
+  password:
+    <passwordSecret>
+----
+Substitute `<usernameSecret>` and `<passwordSecret>` accordingly. The following sample shows basic authentication configured with `remoteWriteAuth` for the `name` values and `user` and `password` for the `key` values, which contain the endpoint authentication credentials:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+        basicAuth:
+          username:
+            name: remoteWriteAuth
+            key: user
+          password:
+            name: remoteWriteAuth
+            key: password
+----
++
+* The following example configures client TLS authentication:
++
+[source,yaml]
+----
+tlsConfig:
+  cert:
+    <certSecret>
+  keySecret:
+    <keySecret>
+----
+Substitute `<certSecret>` and `<keySecret>` accordingly. The following sample shows a TLS authentication configuration using `selfsigned-mtls-bundle` for the `name` values and `ca.crt` for the `ca` `key` value, `client.crt` for the `cert` `key` value, and `client.key` for the `keySecret` `key` value:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+      tlsConfig:
+        ca:
+          secret:
+            name: selfsigned-mtls-bundle
+            key: ca.crt
+        cert:
+          secret:
+            name: selfsigned-mtls-bundle
+            key: client.crt
+        keySecret:
+          name: selfsigned-mtls-bundle
+          key: client.key
+----
+
+. Add write relabel configuration values after the authentication credentials:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+        <endpoint_authentication_credentials>
+        <write_relabel_configs>
+----
++
+For `<write_relabel_configs>` substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint.
++
+The following example shows how to forward a single metric called `my_metric`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote.write.endpoint"
+        writeRelabelConfig:
```

Comment:
I think it should be `writeRelabelConfigs`, see from:
https://github.com/openshift/cluster-monitoring-operator/pull/1308/files#diff-420c938cae65d6e9c4d4e386d768532276257149dbcaf476a90884a29fe36bd4R154

Reply:
Good catch, this is my mistake in the document I provided @bburt-rh with. Fixed in the the gdoc as well.

Reply:
Updated in the doc.